ACCS-365: Payment Services: Stored Cards at Checkout (#821)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/containers/credit-card.mdx

| `creditCardFormRef` | `RefObject<CreditCardFormRef>` | Yes | References the credit card form. Pass \{ current: null \} initially. After rendering, the container sets `current` to a \{ validate: () =&gt; boolean; submit: () =&gt; Promise&lt;void&gt; \} object for programmatic validation and submission. |
| `onSuccess` | `function` | Yes | Executes when the payment flow completes successfully. |
| `onError` | `function` | Yes | Executes when the payment flow aborts due to an error. Receives an object with two properties, \{ name: string, message: string \}, containing the localized error name and message. Both properties are user-facing and translatable through the "PaymentServices.CreditCard.errors" language definitions. **Note:** When omitted, the promise rejects properly so the calling code can catch errors directly. |

> | `getCustomerToken` | `(() => string \| null) \| null` | No | Optional override for the customer auth token getter. When it returns a non-null string, the **Save this card** checkbox is shown and vaulting can be requested on submit. When omitted, the container uses [`getCustomerToken` from initialization](/dropins/payment-services/initialization/). |

Reviewer: ```suggestion
| `getCustomerToken` | `(() => string \| null) \| null` | No | Optional override for retrieving the customer auth token. If it returns a non-null string, the Save this card checkbox is shown and vaulting can be requested on submit. If not provided, the container uses [`getCustomerToken` from initialization](/dropins/payment-services/initialization/). |
```
